### PR TITLE
[#133559651] Memory findWhereEq to take sort as a list

### PR DESCRIPTION
**Pull request by jonajonlee**

### CODE REVIEW

File: lib/memory/index.js [hunk 1]
```diff
@@ -103,19 +103,22 @@ module.exports = function MemoryStore(data_set) {
   })
 
 
-  // _sort :: Sort -> [a] -> [a]
-  const _sort = R.curry((sort, list) => {
+  // _sortToFn :: Sort -> ComparatorFn
+  const _sortToFn = ([key, order]) => R.ifElse(
+    R.equals('desc')
```
Comment (mattsmee) [inline lib/memory/index.js:108]: @jonajonlee if someone didn't spell 'asc' or 'desc' correctly, do we still want to sort?

Comment (jonajonlee) [reply lib/memory/index.js:108]: In this case, it'll assume ascend if they don't specify descend. I think if they're using the sort in general, it's implied that they know it'll be sorted.

Comment (jonajonlee) [reply lib/memory/index.js:108]: In mysql sort, it'll blow up

Comment (mattsmee) [reply lib/memory/index.js:108]: This is only for memory store, so i think it's ok.

**Pull request merged**